scripts/generate-version-info.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/sh
+
+# generates a JSON file containing version information for the software in this AMI
+
+set -o errexit
+set -o pipefail
+
+if [ "$#" -ne 1 ]
+then
+  echo "usage: $0 OUTPUT_FILE"
+  exit 1
+fi
+
+OUTPUT_FILE="$1"
+
+# packages
+yum list installed --quiet | awk '{print $1, $2}' | jq -R 'inputs | split(" ") | {(.[0]):(.[1])}' | jq -s add  | jq '{packages:(.)}' > "$OUTPUT_FILE"
```

Comment:
I tried to run this on my own AMI and it doesn't seem to work correctly. Seems like `awk '{print $1, $2}'` doesn't always result in a valid result. Do you mind checking again?

Reply:
In your output too, you can see the following:
```
"python-backports-ssl_match_hostname.noarch": "",
"3.5.0.1-1.amzn2": "installed",
```

which doesn't seem expected.

Reply:
Ah, thanks; I thought I had addressed this, but `yum` always wraps lines at 80 characters when you pipe. I'll change this to use `rpm`, which has a better API for querying anyway.

Reply:
The `rpm` one-liner turned out to be way easier on the eyes 🙌

Reply:
Yeah, it looks much better now. 👍